Sdl.Community.GroupShareKit.Tests.Integration/Clients/MtProviderClientTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Sdl.Community.GroupShareKit.Clients;
+using Sdl.Community.GroupShareKit.Exceptions;
+using Sdl.Community.GroupShareKit.Models.Response;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Sdl.Community.GroupShareKit.Tests.Integration.Clients
+{
+    public class MtProviderClientTests
+    {
+        [Fact]
+        public async Task GetMtProviderToken()
```

Comment:
Let's try to respect the naming convention for the unit test methods.
This for example should be something like GetMtProviderToken_Throws_IfCredentialsInvalid().